### ISSUE TITLE
Not support uploading LSTM models as add files

### ISSUE THREAD

**Issue by siusoon** [processing/p5.js-web-editor#1346]
<img width="616" alt="Screenshot 2020-03-27 at 15 26 15" src="https://user-images.githubusercontent.com/1446073/77766078-77993000-703f-11ea-9abf-318acbdd3d07.png">

#### Nature of issue?
- Existing feature enhancement

#### Feature enhancement details:
For uploading files (add files), it only check for file size? Currently it doesn't support to upload other LSTM training models (see the dim files as shown on the attached image) I understand this might be due to the issue of file size, but it would be great if the editor allows upload the models for example with size less than 2 MB for example.

**Comment by catarak** [comment]
The web editor is actually checking for the file type, and accepts files up to 5MB. The training files have no file extension, and therefore the web editor doesn't know how to handle them.

Are these training files used with ml5.js? How are they used? Maybe it would also be possible to host them elsewhere and link them to the web editor via URL (if there are no CORS issues).

**Comment by siusoon** [comment]
@catarak Yes it is not a problem to host somewhere but just think would be good if p5.js web editor allows uploading those files as a whole for beginners so they don't have to switch to different places. (yes those files are used with ml5 for the program like charNN: https://github.com/ml5js/ml5-examples/blob/development/p5js/CharRNN/CharRNN_Text/sketch.js)

**Comment by catarak** [comment]
Since training datasets can be pretty big (and a big burden of cost on Processing Foundation), I'm wary of allowing users to upload these files, and maybe it would instead make since to be able to link these files to the web editor via a supplied URL. I'm not sure from a UI/UX perspective how to handle CORS issues, and when it would make sense to instead upload the files to the web editor (like in #387). Some interesting stuff to think about.
